CMakeLists.txt

```
@@ -91,13 +91,13 @@ endif()
 option(PYBIND11_INSTALL "Install pybind11 header files?" ${PYBIND11_MASTER_PROJECT})
 option(PYBIND11_TEST "Build pybind11 test suite?" ${PYBIND11_MASTER_PROJECT})
 option(PYBIND11_NOPYTHON "Disable search for Python" OFF)
-option(PYBIND11_USE_PYPY_GIL "Force use of PYPY GIL acquire/release logic in gil.h" OFF)
+option(PYBIND11_SIMPLE_GIL "Force use of PYPY GIL acquire/release logic in gil.h" OFF)
```

`"Force use of a simple GIL acquire/release logic in gil.h (see #4216 & #4218)"`

As is, the comment suggests we're using something from PYPY, which is not the case.

---

Well, I don't want to put "supports nesting" because no one bothered to add a test for nesting.

How about:

```suggestion
option(PYBIND11_SIMPLE_GIL "Use simpler GIL implementation that does not support disassociation" OFF)
```

---

The suggested revision to the comment looks good to me.